checkly.go

```diff
@@ -736,3 +746,11 @@ func (c *client) apiCall(
 	}
 	return resp.StatusCode, string(res), nil
 }
+
+func withAutoAssignAlertsFlag(url string) string {
+	flag := "autoAssignAlerts=false"
```

Comment:
Isn't there a go util for adding query params already?

Reply:
well, yes we could... but then we'll need to handle errors, 
```
func withAutoAssignAlertsFlag(url string) (string, error) {
         u, err := url.Parse(url")
	if err != nil {
		return "", err
	}
	q := u.Query()
	q.Set("autoAssignAlerts", "true")
	u.RawQuery = q.Encode()
	return u.String()
}
```
and then when we call it:  
```
url, err := withAutoAssignAlertsFlag(url)
if err != nil {
 ....
}

```
so I think for this use-case, it's better to keep it simple since we're sure, there's no errors, & the query param is local constant string

Reply:
sounds good 👍